src/card.css

```
@@ -0,0 +1,99 @@
+.card {
+  display: flex;
+  flex-wrap: wrap;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img {
+  height: 134px;
+  width: 160px;
+  padding: 32px 19px;
```

for internal indents using margin, let's fix in all places
```suggestion
  display: block;
  margin: 0 auto 40;
```